hera_qm/firstcal_metrics.py

```diff
@@ -349,15 +320,8 @@ def __init__(self, calfits_files, use_gp=True):
         self.UVC = UVCal()
         self.UVC.read_calfits(calfits_files)
 
-        if len(self.UVC.jones_array) > 1:
-            raise ValueError('Sorry, only single pol firstcal solutions are '
```

Comment:
Hooray for removing errors because we've added functionality! 😄

Reply:
:tada: